UnitySDK/Assets/ML-Agents/Editor/Tests/StandaloneBuildTest.cs

```diff
@@ -28,9 +28,9 @@ static void BuildStandalonePlayerOSX()
             }
 #else
             var error = buildResult;
-            var isOK = string.IsNullOrEmpty(error);
+            var isOk = string.IsNullOrEmpty(error);
```

Comment:
defined in an `#if` used outside of it.

Reply:
Doh, rider fixed up half of it but not the rest.